Examples/DataRepresentation/Containers/TreeContainer.cxx

```diff
@@ -77,7 +78,7 @@ int main(int, char* [])
 
 
   // Software Guide : BeginCodeSnippet
-  tree->SetRoot(0);
+  tree->SetRoot( 0 );
```

Comment:
Again, for me personally these spaces around an argument are not really necessary. (But I can get used to it.)

Reply:
Same, I got used to it after a while.